### PR TITLE
Sample Code for release of v1.0

### PR DESCRIPTION
**Pull request by RohanShrothrium**
Signed-off-by: rohanshrothrium <shrothriumrohan@gmail.com>
@dexhunter and @yeasy please do check this out.
Let me know if I have to add anything extra.

### CODE REVIEW

**Comment by dexhunter** [comment]
@RohanShrothrium I think we need to update the sample code as in [our tutorial](https://fabric-sdk-py.readthedocs.io/en/latest/tutorial.html#operate-channels-with-fabric-network) (use `asyncio`)

**Comment by yeasy** [comment]
Rerun

**Comment by RohanShrothrium** [comment]
> @RohanShrothrium I think we need to update the sample code as in [our tutorial](https://fabric-sdk-py.readthedocs.io/en/latest/tutorial.html#operate-channels-with-fabric-network) (use `asyncio`)

I've added the asyncio method.
I've also made a small change in the contract class.

File: docs/sample.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    ))
+    print(response)
+
+    # Get Channel configuration
+    responses = loop.run_until_complete(cli.get_channel_config(
+        requestor=org1_admin,
+        channel_name='businesschannel',
+        peers=['peer0.org1.example.com', 'peer1.org1.example.com']
+    ))
+
```
Comment (dexhunter) [inline docs/sample.py:224]: you should also `print` this response

File: docs/sample.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    # Get channel config from orderer
+    response = loop.run_until_complete(cli.get_channel_config_with_orderer(
+        orderer='orderer.example.com',
+        requestor=org1_admin,
+        channel_name=SYSTEM_CHANNEL_NAME,
+    ))
```
Comment (dexhunter) [inline docs/sample.py:230]: same here

**Comment by RohanShrothrium** [comment]
> the last two responses need to be `print`ed as well

Hey! I've made the changes!

**Comment by dexhunter** [comment]
@yeasy What do you think of the sample code?

**Pull request merged**